iceoryx_posh/test/moduletests/test_roudi_process_introspection.cpp

```diff
@@ -65,7 +65,7 @@ class ProcessIntrospection_test : public Test
 
         introspection.send();
 
-        EXPECT_THAT(m_senderPortImpl_mock->deliverChunk, Eq(1));
+        EXPECT_THAT(m_senderPortImpl_mock->deliverChunk, Eq(1u));
```

Comment:
Did we decide to use 1u or 1U? I'm not sure...

Reply:
I'd also like to know.
If I remember right MISRA prefers upper case because lower case 'i' and number '1' might look pretty similar depending on font type.

Reply:
so 50% more typing ... and there goes the productivity

Reply:
According to MISRA-CPP-2.13.4 and AUTOSAR-M2.13.4 all literal suffixes should not contain lowercase characters. This removes potential ambiguity between “1” (digit 1) and  “l” (letter el) for declaring literals (from: https://rules.sonarsource.com/c/RSPEC-818).

Clang-Tidy offers an automatic check for it: https://clang.llvm.org/extra/clang-tidy/checks/readability-uppercase-literal-suffix.html